torch/_refs/nn/functional/__init__.py

```diff
@@ -376,3 +378,37 @@ def gelu(a: TensorLikeType, approximate: str = "none") -> TensorLikeType:
         return a * 0.5 * (1 + torch.erf(a * kAlpha))
     else:
         raise RuntimeError("approximate argument must be either none or tanh.")
+
+
+def prelu(a: TensorLikeType, weight: TensorLikeType) -> TensorLikeType:
```

Comment:
I think this needs a type promotion wrapper? That should resolve the ref's discrepancy with eager when `a` and `weight` have distinct dtypes

Reply:
Shouldn't it already have type promotion from `mul` and `where`? Would it still benefit from the wrapper here?